.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build
+
+on:
+  workflow_call:
+
+permissions:
+  contents: read
+
+jobs:
+  wheel:
+    name: Build Python wheel
+    strategy:
+      matrix:
+        os: ["darwin","linux", "windows"]
+        arch: ["amd64", "arm64"]
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: "1.22"
+          check-latest: true
+          cache: false
+
+      - name: Setup Python
+        uses: actions/setup-python@v5
+
+      - name: Install mage
+        run: go install github.com/magefile/mage@v1.15.0
+
+      - name: Install our package in editable mode
+        run: pip install -e .
+
+      - name: Initialize MLflow repo
+        run: mage repo:init
```

Comment:
Are these steps necessary? I believe they shouldn't be part of the build process.

Reply:
No, they are not, good catch!